incubator/hnc/test/conversion/conversion_test.go

```diff
@@ -94,8 +94,8 @@ var _ = Describe("Conversion from v1alpha1 to v1alpha2", func() {
 		// Convert
 		setupV1alpha2()
 
-		// Only the new value should be present
-		FieldShouldNotContain("ns", "", nsA, ".metadata.annotations", "hnc.x-k8s.io/managedBy:foo")
+		// The new value should be unchanged
```

Comment:
Maybe add "The old annotation is not removed and the new ....."

Reply:
Done